Makefile

```diff
@@ -35,7 +49,6 @@ apply: verify build ## Deploy the kwok controller from the current state of your
 		$(HELM_OPTS) \
 		--set controller.image.repository=$(IMG_REPOSITORY) \
 		--set controller.image.tag=$(IMG_TAG) \
-		--set controller.image.digest=$(IMG_DIGEST) \
```

Comment:
Did this just to make it work on my end, will fix it up so that we don't have to remove this, although it's just a placeholder for a fake value here.